yourgov/components/FormMobileFoodVendorPermit/GettingStartedForm.tsx

```diff
@@ -23,44 +27,100 @@ export function GettingStartedForm() {
 		handleSubmit,
 		formState: { errors },
 	} = useForm<FormSchema>({
-		resolver: yupResolver(formSchema),
+		resolver: zodResolver(formSchema),
+		mode: 'onSubmit',
+		reValidateMode: 'onBlur',
```

Comment:
`revalidate` is a word, isn't it?

Reply:
Yep. That's react-hook-form's bad naming convention